packages/x-components/src/x-modules/popular-searches/store/__tests__/getters.spec.ts

```diff
@@ -19,6 +19,18 @@ describe('testing popular searches module getters', () => {
       resetPopularSearchesStateWith(store, { config: { maxItemsToRequest: 3 } });
       expect(store.getters[gettersKeys.request]).toEqual({ rows: 3, start: 0 });
     });
+
+    it('should return a request object with extra params', () => {
```

Comment:
Instead of creating a new test, in the rest of the PRs we're updating the existing ones:

```typescript
    it('should return a request object with config default values', () => {
      resetPopularSearchesStateWith(store, {
        config: { maxItemsToRequest: 3 },
        params: { catalog: 'es' }
      });
      expect(store.getters[gettersKeys.request]).toEqual<
        SuggestionsRequest & { [key: string]: unknown }
        // TODO - Remove when the facets refactor is completed.
      >({ rows: 3, start: 0, catalog: 'es' });
    });
  });
```

Reply:
But the test also works with this, without casting the equal:

` expect(store.getters[gettersKeys.request]).toEqual({ rows: 3, start: 0, catalog: 'es' });`